src/main/java/org/gridsuite/study/server/StudyException.java

```diff
@@ -22,7 +22,8 @@
         SECURITY_ANALYSIS_RUNNING,
         SECURITY_ANALYSIS_NOT_FOUND,
         NOT_ALLOWED,
-        STUDY_CREATION_FAILED
+        STUDY_CREATION_FAILED,
+        DRIVER_NOT_SUPPORTED
```

Comment:
I don't think it's useful to expose this error to the client code. The client can't do anything about it